perl/buffers.pl

```diff
@@ -1298,8 +1300,7 @@ sub build_buffers
             }
             else
             {
-                my $indent = "";
-                $indent = ((" " x length($buffer->{"number"}))." ") if (($position eq "left") || ($position eq "right"));
+                my $indent = (" " x length($buffer->{"number"}))." ";
```

Comment:
Ahhhh... yes, I see. Thanks again. I'll have to play with that setting and see how to adjust for it.